tests/test_shell_pr_probe_reftable_and_multi_remote.py

```
@@ -0,0 +1,235 @@
+        fi
+        exit 0
+        """
+    )
+
+
+def _run(shell: str, shell_args: list[str], script: Path, body: str, env: dict[str, str]) -> tuple[int, str, str]:
+    command = f'source "$CMUX_TEST_SCRIPT"\n{body}'
+    full_env = dict(os.environ)
+    full_env.update(env)
+    full_env["CMUX_TEST_SCRIPT"] = str(script)
+    result = subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)